### DISCUSSION TITLE
Export GraphQL schema

### DISCUSSION THREAD

**Discussion by melardev** [Kong/insomnia#3580] [Ideas]
Hi, I was using Insomnia for a while now and I am impressed it has so many good features but lacking a very basic one such as exporting the GraphQL schema as SDL, I am not familiar with the source code nor electron apps so I can not afford this very basic challenge, but someone used to it can do it very easily, can someone please add the possibility to export as JSON/SDL? the GraphQL schema is fetched anyway, we just need a way to export it.

**Comment by grubersjoe** [comment]
This seems to be how Altair does it:

https://github.com/imolorhe/altair/blob/a1f6ef92dbead70f5dbca7d4e0fe8c6bda8e6781/packages/altair-app/src/app/modules/altair/services/gql/gql.service.ts#L434

I also would appreciate this feature very much. It's the last thing that makes me sometimes use Altair instead of Insomnia for GQL.

**Comment by dannashirn** [comment]
Any update on this? It would be really helpful to be able to export the schema, specially now that you can import it.
